admin-ui/plugins/auth-server/redux/sagas/authNLdapSaga.js

```diff
@@ -0,0 +1,118 @@
+import { call, put, takeLatest } from 'redux-saga/effects'
+import {
+  getLdapListSuccess,
+  getLdapListFailure,
+  deleteLdapSuccess,
+  deleteLdapFailure,
+} from '../features/authNLdapSlice'
+import { addLdapSuccess, addLdapFailure, editLdapSuccess } from '../features/authNLdapSlice'
+import { updateToast } from 'Redux/features/toastSlice'
+import { DatabaseLDAPConfigurationApi } from 'jans_config_api'
+import { initAudit } from '@/redux/sagas/SagaUtils'
+import { addAdditionalData } from '@/utils/TokenController'
+import { DELETION, FETCH, UPDATE } from '@/audit/UserActionType'
+import { postUserAction } from '@/redux/api/backend-api'
+import { LDAP } from '@/utils/ApiResources'
+
+function deleteLdapApi(configId) {
+  const api = new DatabaseLDAPConfigurationApi()
+  return new Promise((resolve, reject) => {
+    api.deleteConfigDatabaseLdapByName(configId, (error, data) => {
+      if (error) reject(error)
+      else resolve(data)
+    })
+  })
+}
+
+function* deleteLdapSaga({ payload }) {
+  const audit = yield* initAudit()
+  console.log('action in delete ldap saga', payload)
```

Comment:
Remove console.log statements from production code

Debug logging should not be left in production code. These console.log statements on lines 29 and 55 should be removed.


```diff
-  console.log('action in delete ldap saga', payload)
   try {
```

```diff
   try {
-    console.log('action in add ldap saga', action)
-
     addAdditionalData(audit, FETCH, LDAP, { message: action?.payload?.action_message })
```


Also applies to: 55-55